src/main/java/com/synopsys/integration/detect/ApplicationUpdater.java

```diff
@@ -236,6 +318,21 @@ private String findArgument(ListIterator<String> it, String argument) {
         return null;
     }
     
+    private Set<String> findArgumentCommaDelimitedValues(ListIterator<String> it, String argument) {
+        final int equalsIndex;
+        final String delimitedValues;
+        if ((equalsIndex = argument.indexOf("=")) > -1) {
+            delimitedValues = argument.substring(equalsIndex + 1, argument.length());
+            return Arrays.stream(delimitedValues.split("\\,"))
+                .collect(Collectors.toSet());
+        } else if (it.hasNext()) {
+            delimitedValues = it.next();
+            return Arrays.stream(delimitedValues.split("\\,"))
+                .collect(Collectors.toSet());
+        }
+        return null;
+    }
+    
```

Comment:
This method was created to load the comma separated values of an argument, in this case - that of the ignored hosts for proxy argument.

Reply:
I assumed we must already have a method for parsing comma separated values for a Detect property and after a quick glance I believe DetectPropertyConfiguration.getValue() already does this? We should reuse that code if possible